.travis.yml

```diff
@@ -99,68 +99,108 @@ geosx_pangea_build: &geosx_pangea_build
   - gcloud auth activate-service-account --key-file=${GEOSX_GCLOUD_KEY}
   - gsutil cp -a public-read ${GEOSX_BUNDLE} gs://geosx/Pangea2/
 
+draft_script: &draft_script
+  script:
+  # TRAVIS_PULL_REQUEST is false if job is not from a PR
+  - if [[ $TRAVIS_PULL_REQUEST == false ]]; then exit 0; fi;
+  - |
+    is_draft=$(curl -H "Accept: application/vnd.github.v3+json" \
+        https://api.github.com/repos/GEOSX/GEOSX/pulls/$TRAVIS_PULL_REQUEST | \
```

Comment:
Also, I do not really know how this behaves with forks, but I write the comment anyway:

Could we use `TRAVIS_REPO_SLUG` (or `TRAVIS_PULL_REQUEST_SLUG `) instead of hardcoded `GEOSX/GEOSX`?

Reply:
When you open a pull request from a fork into the main GEOSX repo, the PR belongs to GEOSX so I'm relatively certain `TRAVIS_PULL_REQUEST_SLUG` will point to `GEOSX/GEOSX`. That said, if someone opens a PR _inside their own fork_, and they happen to have a Travis account and want to run CI, the right thing to do is indeed to use `TRAVIS_PULL_REQUEST_SLUG` to get the correct endpoint.

Reply:
Thx @klevzoff . Surely could we add the forks of the developers of the GEOSX github organization to our travis plan. Don't know if it's really important though.